docs/advanced/profiling.rst

```diff
@@ -25,7 +39,6 @@ Also, it will write a timeline in Google Chrome trace format
 
 See also this for further information:
 
+* `Optimize TensorFlow performance using the Profiler <https://www.tensorflow.org/guide/profiler#profiler_tools>`__
 * `TensorFlow Profiler and Advisor <https://github.com/tensorflow/tensorflow/blob/b2edbd5a640fb2f50989c5579a4cfe87d1fc675e/tensorflow/core/profiler/README.md>`__
-* `tfprof: TensorFlow Profiler and Beyond <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/tfprof/README.md>`__
```

Comment:
This link and the last one are dead. Does anyone care to fix them / should I just remove them?

Reply:
Yea ok, this can be removed.